Build/Build.cs

```diff
@@ -56,12 +58,37 @@ class Build : RemotionBuild
         var packageJsonPath = ((IBaseBuild)this).Solution.Directory / "Remotion" / "Web" / "Dependencies.JavaScript" / "package.json";
         Assert.FileExists(packageJsonPath);
 
-        var outputFolder = ((IBaseBuild)this).OutputFolder / "Npm" / "remotion.dependencies" / "package.json";
-        var packageJsonContent = packageJsonPath.ReadAllText()
-                .Replace("$version$", ((IBuildMetadata)this).BuildMetadataPerConfiguration.First().Value.Version);
-        outputFolder.WriteAllText(packageJsonContent);
+        var outputPackageJson = ((IBaseBuild)this).OutputFolder / "Npm" / "remotion.dependencies" / "package.json";
+        AddVersionToPackageJson(packageJsonPath, outputPackageJson, ((IBuildMetadata)this).BuildMetadataPerConfiguration.First().Value.Version);
       });
 
+  public override ISbomGeneratorBuilder ConfigureSbomGenerationInfoBuilder (Solution solution)
+  {
+      var version = ((IBuildMetadata)this).GetBaseVersion();
+
+      var semanticVersion = SemanticVersion.Parse(version);
+      var shortenedVersion = $"{semanticVersion.Major}.{semanticVersion.Minor}.{semanticVersion.Patch}";
+
+      // We disregard test projects when generating the sbom already, so these are not required here.
+      var blacklistedProjects = new[]
+                                {
+                                    "Web.Dependencies.Javascript",
+                                    "*.Analyzers*"
+                                };
```

Comment:
Recommend moving the comment into the array (at the end). Otherwise it reads as if the array is the test projects blacklist.